Configurations/unix-Makefile.tmpl

```diff
@@ -555,17 +555,11 @@ libclean:
 		s1=`echo "$$s" | cut -f1 -d";"`; \
 		s2=`echo "$$s" | cut -f2 -d";"`; \
 		s3=`echo "$$s" | cut -f3 -d";"`; \
-		$(ECHO) $(RM) $$s1; {- output_off() unless windowsdll(); "" -}\
-		$(RM) apps/$$s1; \
-		$(RM) test/$$s1; \
-		$(RM) fuzz/$$s1; {- output_on() unless windowsdll(); "" -}\
 		$(RM) $$s1; \
 		if [ "$$s2" != "" ]; then \
-			$(ECHO) $(RM) $$s2; \
 			$(RM) $$s2; \
 		fi; \
 		if [ "$$s3" != "" ]; then \
-			$(ECHO) $(RM) $$s3; \
 			$(RM) $$s3; \
```

Comment:
You've now made `libclean` completely silent...

Reply:
> You've now made `libclean` completely silent...

No, the `$(RM)` command will echo.

Reply:
How do you figure it will do that?

Reply:
Do you have a version of `make` that doesn't echo commands?  I am confused.  Do you think the if statement is too confusing?

Reply:
I believe you haven't really looked.  This is only part of one loooong command, starting at line 553.  That line looks like this:

``` make
	@set -e; for s in $(SHLIB_INFO); do \
```

Now tell me how just those `$(RM)` commands are echoed...

Reply:
I deserve that snark.  Thanks.